dao/src/main/java/com/postal/address/AddressMongoDBApplication.java

```diff
@@ -0,0 +1,36 @@
+package com.postal.address;
```

Comment:
This is just a testing class for reference. Ignore from review. 
Will be cleaned up later.